analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec

```diff
@@ -29,5 +29,6 @@
     <file src="bin\Release\Newtonsoft.Json.dll" target="analyzers" />
     <file src="tools-cs\*.ps1" target="tools\" />
     <file src="logos\sonarsource_64.png" target="images\" />
+    <file src="..\..\..\THIRD-PARTY-NOTICES.txt" target="license\" />
```

Comment:
I may be doing something very wrong here, but if I extract resulting vsix file with 7-zip it has logos directory, but there is no license folder inside.

Reply:
The VSIX is just for debugging during development and it's not relevant.

This is used for the nuget packages we eventually publish on Nuget.org when releasing, and the nugets do have the `license` folder inside - you can download the artifact from : https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=34588&view=artifacts&pathAsName=false&type=publishedArtifacts

Reply:
to generate the nuget locally you can use `nuget pack .\analyzers\src\SonarAnalyzer.Vsix\SonarAnalyzer.CSharp.nuspec` and it will get generated where you run the command from

Reply:
I know it's confusing that the nuspec files which are used in our release process are present in the VSIX folder which is just for debugging, we may want to move them to their own folder as they're important